test/hash_code_test.dart

```diff
@@ -55,9 +55,9 @@ void main() {
 
   test('testHashCodeOptionalEnum', () {
```

Comment:
Let's skip this. I'd rather dartfmt code when we need to touch it for some other reason.